server/middleware/one-time-code-auth/setUpOneTimeCode.ts

```diff
@@ -24,11 +25,7 @@ export default function setUpOneTimeCode(app: Express, oneTimeCodeService: OneTi
   app.use('/oneTimeCode', setUpRequestOneTimeCode(requestOneTimeCodeController))
   app.use('/oneTimeCode', setupOneTimeCodeEmailSent(oneTimeCodeEmailSentController))
   app.use('/oneTimeCode', setupVerifyOneTimeCode(verifyOneTimeCodeController))
-
-  router.get('/oneTimeCode/start-again', (req, res) => res.render('pages/one-time-code-auth/startAgain'))
```

Comment:
It turns out this doesn't work!, so I've had to put it back into a function as it was before